core/dbt/config/renderer.py

```diff
@@ -102,11 +102,12 @@ def should_render_keypath_v2(self, keypath: Keypath) -> bool:
             return False
 
         if first in {'seeds', 'models', 'snapshots', 'seeds'}:
+            keypath_parts = {
```

Comment:
FWIW I think `seeds` is represented twice in the set above

Reply:
This has been here for so long, wow. Fixed!

Reply:
@beckjake looks like there is one more occurence of `seeds` repeated

Reply:
Dang, I missed this comment and only saw the approval. I'll try to catch that next time I'm in here.